tests/docker/Dockerfile

```diff
@@ -248,9 +248,7 @@ RUN python3 -m pip install --force --no-cache-dir -r  /opt/scripts/consumer_offs
 # passes --force so system pip packages can be updated
 COPY --chown=0:0 --chmod=0755 tests/setup.py /root/tests/
 RUN python3 -m pip install --upgrade --force pip && \
-    python3 -m pip install --force --no-cache-dir -e /root/tests/
-
-RUN python3 -m pip install --force-reinstall --no-cache-dir --no-binary :all: confluent-kafka==2.2.0
+    python3 -m pip install --force-reinstall --no-cache-dir --no-binary confluent-kafka -e /root/tests/
```

Comment:
So this will pick up the right version from the setup.py, right? Or will it always install the current version?

Reply:
As I understand it, the "confluent-kafka" there in the command line is strictly an argument to the no-binary option, then the actual package specification is read from root/tests/setup.py as before.

Reply:
Yes, thanks, I had misunderstood was going on here the first time around in part due to the diff. Now that I get it, it looks even better.